pyteal/ast/acct.py

```diff
@@ -0,0 +1,59 @@
+from typing import TYPE_CHECKING
+
+from ..types import TealType, require_type
+from ..ir import Op
+from .expr import Expr
+from .maybe import MaybeValue
+
+if TYPE_CHECKING:
+    from ..compiler import CompileOptions
+
+
+class AccountParam:
+    @classmethod
+    def balance(cls, acct: Expr) -> MaybeValue:
+        """Get the current balance in microalgos an account.
+
+        Args:
+            acct: An index into Txn.accounts that corresponds to the application to check or an address in the Txn.accounts array.
```

Comment:
nit: I think the document for `acct` is not quite accurate, since by https://github.com/algorand/go-algorand/blob/4b1dfed080122c317df7e12e0d83c55871da7ff4/data/transactions/logic/eval.go#L3277, the app's own address is also allowed